cpp/include/cudf/detail/structs/utilities.hpp

```diff
@@ -170,11 +172,11 @@ class flattened_table {
  *         orders, flattened null precedence, alongside the supporting columns and device_buffers
  *         for the flattened table.
  */
-[[nodiscard]] std::unique_ptr<flattened_table> flatten_nested_columns(
+[[nodiscard]] std::unique_ptr<cudf::structs::detail::flattened_table> flatten_nested_columns(
```

Comment:
Removing explicit namespaces feels like a loss of information that is helpful to the reader. Can these be reverted?

Reply:
To be honest I didn't take the time to understand the rules clang-tidy followed for when it liked namespaces and when it didn't. Let me see what I can find about that.

Reply:
Wait hold on, this is adding explicit namespaces, not removing them. Did you like it better with fewer namespaces?

Reply:
😅 I might have read this wrong. I like using explicit namespaces!